packages/fxa-settings/src/components/InputPhoneNumber/index.tsx

```diff
@@ -121,30 +131,15 @@ const InputPhoneNumber = ({
         value={selectedCountry.id}
         className={`bg-transparent border border-grey-200 rounded-md py-2 ps-10 w-[60px] me-2 focus:border-blue-400 focus:outline-none focus:shadow-input-blue-focus ${selectedCountry.classNameFlag} bg-no-repeat bg-[length:1.5rem_1rem] bg-[40%_50%] text-transparent`}
       >
-        {/* Default country is always first */}
-        <option
-          className={`${defaultCountry.classNameFlag} bg-contain`}
-          value={defaultCountry.id}
-        >
-          {defaultCountry.localizedName} ({defaultCountry.code})
-        </option>
-
-        {/* Note, at the time of writing we're using react-dom 18.3 which complains about
-        <hr> inside <select> being invalid, but it is valid. This should be fixed in later
-         versions: https://github.com/facebook/react/issues/27572 */}
-        <hr className="my-1" />
```

Comment:
Did you try hiding this for mobile only to see if that would work alright so we could have the divider for desktop at least? I'm fine with removing it but I know it was a UX preference to have it.

Reply:
I was considering this, but then saw that support for <hr> in <select> is actually still quite low
https://caniuse.com/mdn-html_elements_hr_hr_in_select